### ISSUE TITLE
TPP has comparator output even though it can't do work

### ISSUE THREAD

**Issue by johalun** [TeamPneumatic/pnc-repressurized#1256]
### Describe the bug

As discussed on Discord. The TPP should act like the Refinery when it comes to comparator output. If for any reason it can't do work, there should be no signal output. However, with for example ethanol production the TPP gives comparator output even thought the output tank is full.

### How to reproduce the bug

Place down a TPP with redstone comparator taking output from it. Fill it up with yeast, sugar and ethanol.

### Expected behavior

The comparator should not give redstone signal if the TPP output tank is full with ethanol

### Which Minecraft version are you using?

1.19

### Which version of PneumaticCraft: Repressurized are you using?

1.19.2-4.3.9

**Comment by desht** [comment]
Yeah right now the TPP just checks for a valid recipe. A better "has work" check would also check for valid input materials and space in the output.
